### DISCUSSION TITLE
Are there plans to update / finish the documentation?

### DISCUSSION THREAD

**Discussion by Achder** [pmndrs/p2-es#42] [General]
Hi there,

I just want to clarify up-front: This is not a request and I don't want to pressure anybody. I'm using p2-es for a hobby project and I like it so far. It seems to me it's the most usable 2d physics library with proper typescript support right now. I'm able to work with it the way it is now and the last thing I want is to stress anybody out. Thanks for the work so far to the maintainers! It's a great library.

As to my question: I'd like to know if there are any plans on filling the holes in the documentation. There are some sections that show just a TODO message. I have some prior experience with Box2D that helps fill in some of the gaps. But I'm really not deep into physics simulations in general so I have some trouble understanding all the details. 
For the same reason I'd find it super helpful to document some expected values ranges, for example for stiffness and relaxation. Right now I have to experiment a lot with the values. 

I don't know what the future plan in terms of features are or if it's considered feature complete. Maybe there is some room for polishing the documentation. I'd help out with it but like I said I'm not the deep into the topic. Just a happy user!

**Comment by isaac-mason** [comment]
Glad to hear you're a happy user!

There's no active effort to write those sections, but contributions are very welcome.

I can start by creating issues for the sections that need writing, but I won't have time to write them myself for a while.

**Reply by isaac-mason** [reply]
The easy part is done!

https://github.com/pmndrs/p2-es/issues/43, https://github.com/pmndrs/p2-es/issues/44, https://github.com/pmndrs/p2-es/issues/45, https://github.com/pmndrs/p2-es/issues/46, https://github.com/pmndrs/p2-es/issues/47, https://github.com/pmndrs/p2-es/issues/48
